README.md

```diff
@@ -92,9 +92,10 @@ The general format of the Cloudfuse Linux commands is `cloudfuse [command] [argu
 * `mount all` - Mounts all the containers in an S3 Account or Azure account supported by mount
   - Example: cloudfuse mount all \<mount path> --config-file=\<config file>
 * `mount list` - Lists all Cloudfuse filesystems.
-  - cloudfuse mount list
+  - Example: cloudfuse mount list
 * `unmount` - Unmounts the Cloudfuse filesystem.
-  - Example: cloudfuse unmount \<mount path>
+  - Add "--lazy" (or "-z") flag to use lazy unmount (prevents busy errors)
+  - Example: cloudfuse unmount --lazy \<mount path>
 * `unmount all` - Unmounts all Cloudfuse filesystems.

```

Comment:
Could you add that it also works for unmount all as wel? Maybe just duplicate the part you added for unmount.